File: extract.js
const fs = require("fs");

const filename = "c:\\Users\\samsung\\Downloads\\test3\\index.html";
const html = fs.readFileSync(filename, "utf8");

const regex = /<span class="sentence"[^>]*>(.*?)<\/span>/g;
let match;
const sentences = [];

while ((match = regex.exec(html)) !== null) {
  sentences.push(match[1]);
}

fs.writeFileSync(
  "c:\\Users\\samsung\\Downloads\\test3\\sentences.json",
  JSON.stringify(sentences, null, 2),
  "utf-8",
);

File: extract.py
import re
import json

filename = r"c:\Users\samsung\Downloads\test3\index.html"
with open(filename, "r", encoding="utf-8") as f:
    html = f.read()

sentences = re.findall(r'<span class="sentence"[^>]*>(.*?)</span>', html)
print(json.dumps(sentences, ensure_ascii=False, indent=2))
